Dockerfile

```diff
@@ -52,11 +13,23 @@ RUN adduser --disabled-password \
     --uid ${NB_UID} \
     ${NB_USER}
 
-RUN echo ". ${CONDA_DIR}/etc/profile.d/conda.sh && conda activate base" >> ${HOME}/.bashrc
+RUN /bin/bash -c "\
```

Comment:
Why use bash as intermediary? Can't we run the commands as they are?

Reply:
or maybe it can work now without it? not really sure

Reply:
eh, seem to be working. let me update it without bash -c